src/Containers/RankingTable.tsx

```diff
@@ -73,18 +78,28 @@ export const RankingTable: React.FC<RankingTableProps> = ({
                                 (heading): React.ReactElement => (
                                     <TableHeading>
                                         {heading === 'totalSpent' ? (
-                                            <span
-                                                role="button"
-                                                tabIndex={0}
-                                                onKeyDown={(): void =>
-                                                    setIsAscending(!isAscending)
-                                                }
-                                                onClick={(): void =>
-                                                    setIsAscending(!isAscending)
-                                                }
-                                            >
-                                                {heading.toUpperCase()}
-                                            </span>
+                                            <TotalSpentDiv>
+                                                <span
+                                                    role="button"
+                                                    tabIndex={0}
+                                                    onKeyDown={(): void =>
```

Comment:
Why is there two btw?

Reply:
Should I not pass this function to the onKeyDown event?

Reply:
Just not sure why you are calling the function twice? isn't onClick enough?

Reply:
https://reactnative.dev/docs/pressable
Look at the diagram this might explain things more :)

Reply:
Okay thanks :) but I was getting an eslint error "Visible, non-interactive elements with click handlers must have at least one keyboard listener." [https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/click-events-have-key-events.md](url) for accessibility, but I can ignore the error

Reply:
Yes but you have to specify the key that was down to catch the correct key. This is more advanced we can visit later since accessibility is very complex to get right 👍 

For now just ignore the error.

Reply:
Sounds good. Can I merge this into v2 without having the dropdown finished?

Reply:
@arkiis Why not just import the existing dropdowns from React-UI?

Reply:
UI Layout:
Header: Top 10 Items (HeaderTitle: string as a Prop)
DropDown
Table.